docs/HACKING.md

```diff
@@ -35,6 +35,11 @@ You will also need to rerun this after the dependency changes in our
 Otherwise, you can use e.g. `sudo dnf builddep rpm-ostree` to get
 the rest of the build dependencies.
 
+### Installing dependencies: testing
+
+In addition to the build dependencies mentioned above, the tests from `make
+check` require `cpio`, too.
```

Comment:
We actually have a script to install test dependencies. We should mention it here. So e.g. instead of this, how about

````suggestion
In addition to the build dependencies mentioned above, the tests require various additional packages. You can install them using:

```
$ ./ci/install-test-deps.sh
```
````

?

Would be good to verify that it installs `cpio`, and if not you can add it to `testdeps.txt`.

Reply:
Done, cpio is in fact not pulled in by any of the packages in `testdeps.txt`. What also makes me wonder is that `cosa` doesn't seem to be present in the repos? At least I still can't `make vmcheck` after installing the test deps, and I can't find cosa via dnf either. Is there a deeper magic behind this?

Reply:
Thanks, good point re. cosa. `make vmcheck` needs kola, which can be found in the cosa container.

OK I think the cleanest is to
1. enhance the buildroot image to include the test deps
2. recommend using the buildroot image here for building and testing

I did 1 in https://github.com/coreos/fedora-coreos-config/pull/1808. Do you want to do 2 here?

Reply:
2 done. I took the liberty to do a bigger rewrite of the docs here because I found it pretty confusing to understand what needs to be done. I hope that's ok!